integration-tests/redis-client/src/main/resources/application.properties

```
@@ -7,3 +7,5 @@ quarkus.redis.parameter-injection.hosts=redis://localhost:6379/2
 quarkus.redis.instance-client.hosts=redis://localhost:6379/5
 # use DB 3
 quarkus.redis.provided-hosts.hosts-provider-name=test-hosts-provider
+
+quarkus.redis.load-script=starwars.redis
```

```suggestion
quarkus.redis.load-script=starwars.redis

```
np: adding a new line